juriscraper/scotus/scotus_docket_report.py

```diff
@@ -0,0 +1,413 @@
+                len(prefix_match.group(0)) :
+            ].split(",")
+            return self._clean_parts(
+                [f"{prefix}{n.strip('-')}" for n in numbers if n.strip("-")]
+            )
+
+        # Case 4: Comma-separated items with prefixes already correct: "33094-CW, 33095-CW"
+        if self.LOWER_COURT_PATTERNS["comma_correct_prefix"].match(raw_text):
+            return raw_text.split(",")
+
+        # Case 5: Mixed slash-separated values with a shared prefix: "98-16950/17044/17137"
+        if self.LOWER_COURT_PATTERNS["slash_shared_prefix"].match(raw_text):
+            prefix = raw_text.split("-")[0]
+            parts = raw_text.split("/")
+            return self._clean_parts(
+                [f"{prefix}-{p.split('-')[-1]}" for p in parts]
+            )
+
+        # Case 6: Ampersand-separated with prefixes or 'See also'-separated with prefixes: "95-56639 & 96-55194" or "95-56639 See also 96-55194"
+        if "&" in raw_text:
+            return raw_text.replace(" & ", ",").split(",")
+        if "See also" in raw_text:
+            return raw_text.replace(" See also ", ",").split(",")
+
+        # Case 7: Preserve specific formats: "CR-99-1140", "1998-CA-0022039-MR"
+        if any(
+            p.match(raw_text)
+            for p in self.LOWER_COURT_PATTERNS["special_formats"]
+        ):
+            return raw_text.split(",")
+
+        # Case 8: Range expansion: "97-1715/98-1111 to 1115" or "97-1715/1111 to 1115"
+        if re.search(r"\d+/\d+(-\d+)? to \d+", raw_text):
+            parts = raw_text.split("/")
+            cleaned = [parts[0]]
+            range_part = parts[1]
+
+            if " to " in range_part:
+                start, end = range_part.split(" to ")
+                if "-" in start:
+                    sprefix, snum = re.match(r"(\d+)-(\d+)", start).groups()
+                    cleaned.append(f"{sprefix}-{snum}")
+                    end_num = int(re.match(r"(\d+-)?(\d+)", end).groups()[-1])
+                    cleaned.extend(
+                        f"{sprefix}-{i}"
+                        for i in range(int(snum) + 1, end_num + 1)
+                    )
+                else:
+                    prefix = parts[0].split("-")[0]
+                    s, e = map(int, range_part.split(" to "))
+                    cleaned.extend(f"{prefix}-{i}" for i in range(s, e + 1))
+            else:
+                cleaned.append(
+                    f"{parts[0].split('-')[0]}-{range_part.strip()}"
+                )
+
+            return self._clean_parts(cleaned)
+
+        # Case 9: Various separators with inherited prefixes: "98-4033;-4214;-4246"
+        if self.LOWER_COURT_PATTERNS["sep_inherited_prefix"].match(raw_text):
+            parts = re.split(r"[;/,-]", raw_text)
+            prefix = parts[0].split("-")[0]
+            return self._clean_parts(
+                [f"{prefix}-{p.strip('-')}" for p in parts]
+            )
```

Comment:
The condition for this `if` statement is the same as the one used in Case 2. Since Case 2 is checked first and has similar logic, this code block is likely unreachable.